etc/esp-idf/CMakeLists.txt

```diff
@@ -0,0 +1,159 @@
+set(SRCS
+    $ENV{OPENMRNPATH}/src/dcc/dcc_constants.cxx
+    $ENV{OPENMRNPATH}/src/dcc/DccDebug.cxx
+    $ENV{OPENMRNPATH}/src/dcc/Defs.cxx
+    $ENV{OPENMRNPATH}/src/dcc/LocalTrackIf.cxx
+    $ENV{OPENMRNPATH}/src/dcc/Loco.cxx
+    $ENV{OPENMRNPATH}/src/dcc/Packet.cxx
+    $ENV{OPENMRNPATH}/src/dcc/RailcomBroadcastDecoder.cxx
+    $ENV{OPENMRNPATH}/src/dcc/RailCom.cxx
+    $ENV{OPENMRNPATH}/src/dcc/RailcomDebug.cxx
+    $ENV{OPENMRNPATH}/src/dcc/SimpleUpdateLoop.cxx
+    $ENV{OPENMRNPATH}/src/dcc/UpdateLoop.cxx
+    
+    $ENV{OPENMRNPATH}/src/executor/AsyncNotifiableBlock.cxx
+    $ENV{OPENMRNPATH}/src/executor/Executor.cxx
+    $ENV{OPENMRNPATH}/src/executor/Notifiable.cxx
+    $ENV{OPENMRNPATH}/src/executor/Service.cxx
+    $ENV{OPENMRNPATH}/src/executor/StateFlow.cxx
+    $ENV{OPENMRNPATH}/src/executor/Timer.cxx
+
+    $ENV{OPENMRNPATH}/src/freertos_drivers/esp32/Esp32HardwareI2C.cxx
+    $ENV{OPENMRNPATH}/src/freertos_drivers/esp32/Esp32HardwareTwai.cxx
+    $ENV{OPENMRNPATH}/src/freertos_drivers/esp32/Esp32Ledc.cxx
+    $ENV{OPENMRNPATH}/src/freertos_drivers/esp32/Esp32SocInfo.cxx
+    $ENV{OPENMRNPATH}/src/freertos_drivers/esp32/Esp32WiFiManager.cxx
+
+    $ENV{OPENMRNPATH}/src/openlcb/AliasAllocator.cxx
+    $ENV{OPENMRNPATH}/src/openlcb/AliasCache.cxx
+    $ENV{OPENMRNPATH}/src/openlcb/BLEAdvertisement.cxx
+    $ENV{OPENMRNPATH}/src/openlcb/BLEService.cxx
```

Comment:
These BLE files have not been merged back to master yet. I tested this on a branch with BLE, which is why they are here. We need to also add the src/ble subdirectory files to the SRCS list.

Reply:
This is resolved by merging the BLE basics branch into master and then merging master into this branch.